packages/dd-trace/src/plugins/util/ci.js

```diff
@@ -81,6 +81,13 @@ function filterSensitiveInfoFromRepository (repositoryUrl) {
   }
 }
 
+function getTestTypeFromFramework (testFramework) {
+  if (testFramework === 'playwright' || testFramework === 'cypress') {
+    return 'browser'
+  }
+  return 'test'
+}
+
```

Comment:
let's move this function to `packages/dd-trace/src/plugins/util/test.js` so we don't have to export/import it